src/features/defi/providers/fox-farming/components/FoxFarmingManager/Deposit/components/Confirm.tsx

```diff
@@ -128,6 +134,31 @@ export const Confirm: React.FC<StepComponentProps & { accountId: AccountId | und
       if (!txid) throw new Error('Transaction failed')
       dispatch({ type: FoxFarmingDepositActionType.SET_TXID, payload: txid })
       onOngoingFarmingTxIdChange(txid, contractAddress)
+
+      if (contractAddress === ETH_FOX_STAKING_EVERGREEN_CONTRACT && asset && accountId) {
```

Comment:
Why for Evergreen only?

Reply:
re: this one, since this is already in the FOX Farming deposit DeFi modal, I think it doesn't hurt to remove the check altogether, but not required as part of this PR if you want to keep it for now